<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Основные</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Пути к директориям</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Приложений</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLineEdit" name="softFolderEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="softChoiseFolderBtn">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="updateFolderlabell">
            <property name="text">
             <string>Обновлений</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="updatefolderEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="updateChoiseFolderBtn">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Драйверпак</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLineEdit" name="DRPFolderEdit"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="recursiveSearchBox">
         <property name="text">
          <string>Разрешить рекурсивный поиск в подпапках</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="useAutoBatBox">
         <property name="text">
          <string>Использовать файл auto.bat как индикатор наличия программы в папке</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="useAutoCmdBox">
         <property name="text">
          <string>Использовать файл auto.cmd как индикатор наличия программы в папке</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="useInfoTxtBox">
         <property name="text">
          <string>Использовать файл info.txt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="useIconsFromExeBox">
         <property name="text">
          <string>Разрешить получение значков приложения из exe  файлов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ignoreDuplicateBox">
         <property name="text">
          <string>Если в папке несколько файлов-индикаторов использовавть только 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="infotxtIsPreferBox">
         <property name="text">
          <string>Info.txt приоритетнее bat и cmd файлов (рекомендуется)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ignoreBachOnlyBox">
         <property name="text">
          <string>Если в папке только файлы auto.cmd/auto.bat без exe или msi считать папку группой</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="osNameFolderBox">
         <property name="text">
          <string>Обрабатывать папки с кодовыми названиями OS (рекомендуется)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="addPathCheckBox">
         <property name="text">
          <string>Разрешить неявную подстанку путей (небезопасно)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>8</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Внешний вид</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="ShowInfoBlockBox">
         <property name="text">
          <string>Показывать информационный блок в главном окне</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showProgressBarBox">
         <property name="text">
          <string>Использовать прогрессбар</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Базовая ширина блока программы</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QSpinBox" name="AppBoxWidthSpin">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>250</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="value">
              <number>300</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Базовая высота развернутого блока</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QSpinBox" name="AppBoxHSpin">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>50</number>
             </property>
             <property name="maximum">
              <number>5000</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>350</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="consoleBox">
         <property name="text">
          <string>Показывать консоль приложения</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="debugBox">
         <property name="text">
          <string>Режим отладки</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
